src/modules/users/users.service.ts

```diff
@@ -25,7 +25,7 @@ export class UsersService {
   }
 
   async update(userDto: UserDto): Promise<Query<any>> {
-    return await this.userModel.updateOne({ _id: userDto._id }, userDto);
+    return await this.userModel.updateOne({ _id: userDto._id }, userDto, {runValidators: true});
```

Comment:
`{runValidators: true}` nice finding! Thanks!

Reply:
That one was super hard to find.  I'm honestly shocked that the default setting is to skip validation on update; I got burned hard by it 🙁